pages/user.js

```diff
@@ -49,6 +202,22 @@ class User extends Component {
                                 src={isLoggedIn ? picture : ''}
                             />
                         </div>
+                        <div>
+                            <p>Online Users</p>
+                            {onlineUsers.map(value => (
+                                <li key={value.id}>
+                                    <p>{value.name}</p>
+                                    <Button
+                                        type="Primary"
+                                        onClick={() =>
+                                            this.challengeUser(value)
+                                        }
+                                    >
+										Challenge
+                                    </Button>
+                                </li>
+                            ))}
```

Comment:
```
{
    onlineUsers.map(...)
}
```

Reply:
I didnt quite get this
Do you mean making it a one liner like
` {
     onlineUsers.map(value => displayOnlineUser(value))
}`

Eslint is not going to allow me do this it removes the whitespace